src/libNode/Node.h

```diff
@@ -283,7 +281,7 @@ class Node : public Executable, public Broadcastable
     bool ProcessVCBlock(const vector<unsigned char>& message,
                         unsigned int cur_offset, const Peer& from);
 
-#ifndef IS_LOOKUP_NODE
+    //#ifndef IS_LOOKUP_NODE -- normal nodes specific
```

Comment:
Can probably just remove these lines, to be consistent with the other header file changes?